client/components/Main/ActionsSubnavDropdown.tsx

```diff
@@ -23,11 +23,11 @@ const ActionsSubnavDropdownComponent = (props) => {
 
     if (props.privileges[PRIVILEGES.MANAGE_CONTENT_PROFILES]) {
         items.push({
-            label: gettext('Manage planning profiles'),
+            label: gettext('Manage planning profile'),
```

Comment:
seems like those labels are used in e2e
https://github.com/superdesk/superdesk-planning/blob/develop/e2e/cypress/support/planning/manageContentProfiles.ts#L9